dist/locales/zh-HK.json

```diff
@@ -128,20 +128,20 @@
   "impac.widget.settings.limit_entries.display": "你好",
   "impac.widget.settings.limit_entries.entries": "你好",
 
-  "impac.dashboard_selector.delete_widget.description": "你好 你好 你好 你好 你好 你好 你好 你好 你好 你好?",
+  "impac.dashboard_selector.delete_widget.description": "你好 你好 你好 你好 你好 你好 你好 你好 你好 {name}?",
```

Comment:
Not like it's hard ;P , and the rest of the file is implementing the variables so meh